app/scripts/controllers/app-state.js

```diff
@@ -74,6 +74,7 @@ export default class AppStateController extends EventEmitter {
       // States used for displaying the changed network toast
       switchedNetworkDetails: null,
       switchedNetworkNeverShowMessage: false,
+      shouldShowAggregatedBalancePopover: true, // by default user should see popover;
```

Comment:
Would be cool one day if we could scope all of these "onboarding" related state variables within a single object, like `onboardingState: {}` to make it easier to understand which vars are related to onboarding vs what's related to the root level metamask state (unrelated to onboarding)